crates/topos-tce-broadcast/src/lib.rs

```diff
@@ -124,38 +101,28 @@ impl ReliableBroadcastClient {
         config: ReliableBroadcastConfig,
         local_peer_id: String,
         storage: StorageClient,
-        network_client: topos_p2p::Client,
     ) -> (Self, impl Stream<Item = Result<ProtocolEvents, ()>>) {
         let (subscriptions_view_sender, subscriptions_view_receiver) =
             mpsc::channel::<SubscriptionsView>(2048);
         let (event_sender, event_receiver) = broadcast::channel(2048);
-        let (broadcast_commands, command_receiver) = mpsc::channel(2048);
+        let (broadcast_commands, command_receiver) = mpsc::channel(*constant::COMMAND_CHANNEL_SIZE);
```

Comment:
Not 100% part of this PR, but the namings of the channels are a bit confusing. Should we unify the names here? We have `event_sender` above, and `broadcat_commands` here. Above we actually have a `broadcast::channel`, whereas here we have `mpsc::channel`. So I would just go for `command_sender` and add a comment above it for what this is used?